python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -105,3 +82,73 @@ def getExtensionName(md):
         @param md (PropertySet) FITS header metadata
         """
         return md.get('EXTNAME')
+
+#     def getDestination(self, butler, info, filename):
```

Comment:
`getDestination()` can stay here. If you look at obs_decam/config/ingestCalibs.py, the first two line are retargeting the IngestCalibTask's parse task to this decam-specific one. This function will effectively replace the simplified version in pipe_tasks (which I left a comment about over there). This file will then only contain a `DecamCalibsParseTask`, and I don't think any other classes.